projects/plugins/jetpack/modules/widgets/milestone/class-milestone-widget.php

```diff
@@ -115,44 +116,9 @@ public static function styles_template() {
 <style>
 .milestone-widget {
 	margin-bottom: 1em;
```

Comment:
```suggestion
```

One more line to remove and I think we're good. :)

Reply:
oops! Missed that one. Fixed in: https://github.com/Automattic/jetpack/pull/21904/commits/dacc296cb993b37911e325b005eb87786744cc47. Cheers! :)